src/pages/home/videoList.js

```diff
@@ -40,7 +53,7 @@ export default function VideoList() {
   };
 
   return (
-    <div>
+    <Wrapper>
       {videos.map((video) => {
         return (
           <Video
```

Comment:
can add a loader because you are setting the videos state inside useEffect, so for the very first render I think it will give an error, since you are trying to map over an empty array.

Reply:
empty array will not run the map, I added conditional map